src/_internals/item/ItemRadioCheckbox.style.tsx

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components'
+
+import { color, inputBorderSize } from '../../_utils/branding'
+import { Item } from './index'
+
+export const StyledItemRadioCheckbox = styled(Item)`
+  & {
+    cursor: ${props => (props.disabled ? 'default' : 'pointer')};
+    border: ${inputBorderSize.focus} solid transparent;
+  }
+
+  &.focus-visible {
+    outline: ${inputBorderSize.focus} solid ${color.blue};
+  }
+
+  & input {
+    position: absolute;
+    clip: rect(0, 0, 0, 0);
+  }
+
+  &.focus {
```

Comment:
I'll rename it `.focus-visible` as it's only applied when keyboard nav is used, provided by the `useFocusVisible` and it's named like this in other component and in Kairos.

Reply:
Yes as discussed I remove this class, the same rules are already applied on `.focus-visible` above, and I'll change it in components for consistency.